yarn_api_client/hadoop_conf.py

```diff
@@ -62,34 +57,20 @@ def get_resource_manager_host_port():
 def get_jobhistory_host_port():
```

Comment:
Seems like `get_jobhistory_endpoint()` is more fitting now.

Reply:
Adjusted